.github/actions/build-and-push-octopus/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and Push Octopus Container
+run-name: Build and Push Octopus Container by @{{github.actor}}
+description: Build and push Octopus container image to the registry
+inputs:
+  acr_registry:
+    description: Azure Container Registry to push the image to
+    required: true
+  acr_username:
+    description: Azure Container Registry username
+    required: true
+  acr_password:
+    description: Azure Container Registry password
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+
+    - name: Log in to the Azure Container registry
+      uses: docker/login-action@v3.1.0
+      with:
+        password: ${{ inputs.acr_password }}
+        registry: ${{ inputs.acr_registry }}
+        username: ${{ inputs.acr_username }}
+
+    - name: Set variables
+      working-directory: ./octopus
+      shell: bash
+      run: |
+        VER=$(cat VERSION)
+        echo "VERSION=$VER" >> $GITHUB_ENV  
+
+    - name: Build and push Docker image
+      uses: docker/build-push-action@v5
+      with:
+        context: ./octopus
+        file: ./octopus/dockerfile
+        push: true
+        tags: |
+          ${{ inputs.acr_registry }}/dibbs-cloud/octopusdeploy:${{ env.VERSION }}
```

Comment:
wait, this works?  getting `env.VERSION` from a previous step? :eyes: If so, great, I don't think I knew that, I've always found myself using the id trick like in [this file](https://github.com/CDCgov/dibbs-cloud/blob/main/.github/workflows/checkForChanges.yml)

Reply:
It worked for me! 😅 At least, it _seemed_ to work for me.